src/components/Referrals/referralsHelper.js

```diff
@@ -69,6 +76,39 @@ export const tierDiscountInfo = {
   2: 10,
 };
 
+export const totalRebateInfo = {
+  0: 10,
+  1: 20,
+  2: 25,
+};
+
+export function getDiscountSharePercentage(tierId, discountShare) {
+  if (!tierId) return;
+  if (!discountShare || discountShare?.eq(0)) return tierDiscountInfo[tierId];
+  const decimals = 4;
+
+  const totalDiscount = totalRebateInfo[tierId];
+  const discountPercentage = bigNumberify(totalDiscount)
+    .mul(discountShare)
+    .mul(Math.pow(10, decimals))
+    .div(BASIS_POINTS_DIVISOR);
+
+  return removeTrailingZeros(formatAmount(discountPercentage, decimals, 3, true));
+}
+
+export function getRebateSharePercentage(tierId, discountShare) {
+  if (!tierId) return;
+  if (!discountShare || discountShare?.eq(0)) return tierRebateInfo[tierId];
+  const decimals = 4;
+
+  const totalDiscount = totalRebateInfo[tierId];
+  const discountPercentage = bigNumberify(totalDiscount)
+    .mul(MAX_DISCOUNT_SHARE - discountShare)
+    .mul(Math.pow(10, decimals))
+    .div(BASIS_POINTS_DIVISOR);
+  return removeTrailingZeros(formatAmount(discountPercentage, decimals, 3, true));
+}
```

Comment:
these two functions are almost identical, the only difference is default value returned and first multiplier for `totalDiscount`). i think it would be better to refactor it into single function

Reply:
Refactored!